stock_release_channel/models/stock_move.py

```diff
@@ -16,7 +16,9 @@ def release_available_to_promise(self):
         # As moves can be merged (and then unlinked), we should ensure
         # they still exist.
         moves = self.exists()
-        moves.picking_id.assign_release_channel()
+        company = self.env.company
```

Comment:
There is a company field on the move, you need to use that one
```suggestion
        for company, imoves in groupby(moves, key=itemgetter("company_id"))
            if company.recompute_channel_on_pickings_at_release:
                moves = self.env['stock.move'].concat(*imoves)
                moves.picking_id.assign_release_channel()
```

Reply:
Thank @jbaudoux , I updated the PR